packit/local_project.py

```diff
@@ -135,6 +135,18 @@ def ref(self) -> Optional[str]:
             return self._get_ref_from_git_repo()
         return None
 
+    @property
+    def commit(self) -> str:
```

Comment:
nit: can we rename this to something more descriptive? i.e. `commit_hash`, `commit_sha`, `commit_hexsha` etc.

Reply:
fixed

During the renaming, I've found that it was a bad name for the property since it conflicts with the `GitCommit` objects.